README.md

```diff
@@ -106,6 +106,36 @@ You can theme the borders. Here is an card example.
 
 ![Example with card edges](/images/card.png)
 
+Reasons
+----
+
+**Why use &lt;twitter-status&gt; instead of Twitter's embedded tweets?**
+
+- Open source - If you don't like something about it you can customize it to fit your exact needs.
+- Lightweight
+  - A minimal &lt;twitter-status&gt; [example](https://twitter-e9454.firebaseapp.com/twitter-status) comes in at **33KB** before compression. ([With more to save](https://github.com/abraham/twitter-status/issues/79))
+  - A minimal [Twitter Embed](https://developer.twitter.com/en/docs/twitter-for-websites/embedded-tweets/overview.html) [example](https://twitter-e9454.firebaseapp.com/oembed) comes in at **121KB** before compression.
+- Security - You can perform a security audit of &lt;twitter-status&gt; and know exactly what you are shipping. You don't have to worry about loading Twitter's JavaScript.
```

Comment:
Good point, in the light of https://hackernoon.com/im-harvesting-credit-card-numbers-and-passwords-from-your-site-here-s-how-9a8cb347c5b5